### ISSUE TITLE
Vectorization

### ISSUE THREAD

**Issue by Yakov76** [arkypita/LaserGRBL#367]
Hi guys,
How do I make one vector instead of two? Is there such an opportunity for this program? Files attached...
Thanks !
Yakov
![542](https://user-images.githubusercontent.com/31748748/40628422-0423b3de-62d6-11e8-84e9-40778f12a96d.jpg)
![836](https://user-images.githubusercontent.com/31748748/40628429-10b656c4-62d6-11e8-983b-d84a5d85385d.jpg)

**Comment by arkypita** [comment]
Duplicated: please read developmemt roadmap and closed issues before posting.

**Comment by Yakov76** [comment]
I looked through, but did not find ... Can I get a link on this?

**Comment by arkypita** [comment]
https://github.com/arkypita/LaserGRBL/issues/173

**Comment by arkypita** [comment]
Hi all!

I have released a new version of LaserGRBL that include **centerline vectorization**.
This feature was requested by a lot of user and complete LaserGRBL with a "must have features" for all who need a simple way to engrave images.

This release was released as a pre-release so it is not automatically updated. 
To those who want to test it I recommend downloading the installer from: [here](https://github.com/arkypita/LaserGRBL/releases/download/v3.0.0/install.exe)
